roles/openshift_facts/library/openshift_facts.py

```diff
@@ -809,22 +809,6 @@ def set_deployment_facts_if_unset(facts):
             elif deployment_type in ['enterprise']:
                 service_type = 'openshift'
             facts['common']['service_type'] = service_type
-        if 'config_base' not in facts['common']:
```

Comment:
Probably also want to update the docstring for `set_deploiyment_facts_if_unset` to remove config_base as well, otherwise lgtm.